AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.m

```diff
@@ -202,28 +202,61 @@ - (void)removeAuthToken:(nullable NSString *)authToken {
     return (NSArray<MSAuthTokenInfo *> *)self.authTokenHistoryArray;
   }
   NSData *encryptedData = [MS_USER_DEFAULTS objectForKey:kMSAuthTokenHistoryKey];
+
```

Comment:
Our convention of in SDK is to only add a blank newline before comments. Let's remove all the unnecessary newlines for consistency.